Deploy-CAM.ps1

```diff
@@ -2519,6 +2526,59 @@ function Confirm-ModuleVersion()
     return $true
 }
 
+# Create a custom role for CAM with necessary permissions
+# Currently it is the based off of the Contributor Role excpept we remove Storage Access and VM Disk Blob Read Access
```

Comment:
I think the implication here is that the account cannot access any storage account (unless it gets a token from the keyvault...) but really that means that we ONLY support managed storage now since it can no longer create the disks for unmanaged storage?

Reply:
Yeah, we can't do anything with storage accounts after this change including making VHDs. Now that I think about it, this will probably make our VHD template stuff break...

Reply:
Good point on the template stuff - yeah we can't break that.

Reply:
Yeah, VHD templates are broken :(

Reply:
I think it's okay to have blob read access in the resource groups by default as long as we don't have disk read access in the resource groups.